docs/Solidity_Support.md

```diff
@@ -4,14 +4,21 @@ title: Solidity Support
 custom_edit_url: https://github.com/OffchainLabs/arbitrum/edit/master/docs/Solidity_Support.md
 ---
 
-Arbitrum Rollup allows you to deploy a set of Solidity contracts as on a trustless layer 2 sidechain. Arbitrum supports almost all solidity code as expected with a few exceptions.
+# TODO: We talk about Solidity, but should we change to EVM?
```

Comment:
Yes, I think we want to say we support EVM, therefore Solidity support comes for free.